pkg/build/api/v1/types.go

```diff
@@ -402,6 +402,11 @@ type SourceBuildStrategy struct {
 
 	// ForcePull describes if the builder should pull the images from registry prior to building.
 	ForcePull bool `json:"forcePull,omitempty"`
+
+	// disableImplicitBuild prevents source-to-image from performing a Docker build operation
```

Comment:
I can't think of a single reason, other than security, that a user would
ever select DisableImplicitBuild themselves.

On Thu, Apr 28, 2016 at 2:14 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/build/api/v1/types.go
> <https://github.com/openshift/origin/pull/8664#discussion_r61476003>:
>
> > @@ -402,6 +402,11 @@ type SourceBuildStrategy struct {
> >
> >  	// ForcePull describes if the builder should pull the images from registry prior to building.
> >  	ForcePull bool `json:"forcePull,omitempty"`
> > +
> > +	// disableImplicitBuild prevents source-to-image from performing a Docker build operation
>
> yeah, @gabemontero <https://github.com/gabemontero> already has a pull
> open that's updating them all though:
>
> https://github.com/openshift/origin/pull/8477/files#diff-43dd40b4c2b85b388a3d9116729032e4L14
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8664/files/4251157e4e6c6d4b865d530bb0de37eb6fd75b3a#r61476003>
>